<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Unos</class>
 <widget class="QDialog" name="Unos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unos</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>331</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Izbor akcije</string>
   </property>
   <widget class="QRadioButton" name="radioButtonUpis">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Upis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBrisanje">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Brisanje</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonUpisi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Upisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonObrisi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Obrisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>281</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelSifra">
      <property name="text">
       <string>Sifra</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditSifra">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelMesto">
      <property name="text">
       <string>Mesto</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditMesto"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelDatum">
      <property name="text">
       <string>Datum</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="dateEditDatum">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>4</month>
        <day>18</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>dddd , MMMM    dd . yyyy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
